homeassistant/components/fastdotcom/sensor.py

```diff
@@ -52,10 +51,8 @@ async def async_added_to_hass(self) -> None:
                 self.hass, DATA_UPDATED, self._schedule_immediate_update
             )
         )
-
-        if not (state := await self.async_get_last_state()):
-            return
-        self._attr_native_value = state.state
+        # Initial run to get data
+        self.update()
```

Comment:
This would block it for completion as it takes a long time.
I think we should go back to restoring the previous value and let it update on it's regular cycle